src/components/Countdown/Countdown.spec.js

```diff
@@ -0,0 +1,31 @@
+import { screen, render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import Countdown from './Countdown'
+
+jest.useFakeTimers()
+
+describe('Countdown', () => {
+  it('renders the countdown display and one button', () => {
+    render(<Countdown />)
+    expect(screen.getByText('25:00')).toBeInTheDocument()
+    expect(screen.getAllByRole('button')).toHaveLength(1)
+  })
+
+  it('calls setTimeout by clicking the button', () => {
+    render(<Countdown />)
+    expect(setTimeout).toHaveBeenCalledTimes(0)
+    userEvent.click(screen.getByRole('button'))
+    expect(setTimeout).toHaveBeenCalledTimes(1)
+    expect(setTimeout).toHaveBeenLastCalledWith(expect.any(Function), 1000)
+  })
+
+  it('does not call setTimeout function by clicking the button a second time, but does on a third time', () => {
+    render(<Countdown />)
+    userEvent.click(screen.getByRole('button'))
+    userEvent.click(screen.getByRole('button'))
```

Comment:
This looks like a duplicate, but i guess it is needed to simulate a second click? Wonder, if there is another solution to this, which is more comprehensible on first sight.

Reply:
Unfortunately, I couldn't find a shorter solution. Good Point however. I will have a deeper look during the next iteration